src/Authentication/login.js

```diff
@@ -55,9 +58,29 @@ const Login = ({ history }) => {
     }
   }
 
+
+
   if (currentUser!==null && currentUser!==false) {
+
+      getUserAPI(currentUser).then((entry) => {
+        console.log("We got the current user and it is ")
+        console.log(entry)
+        if (entry != null && entry.length == 0){
+          let user = {
+            userId: currentUser.uid,
+            name: "",

```

Comment:
Should we default name field to empty string? Is there a way to extract this information?

Reply:
Currently, no way to extract this field.